#! /usr/bin/env python

PACKAGE='jsk_perception'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("nmixtures", int_t, 0, "the number of gaussian mixture", 3, 1, 10)
gen.add("detect_shadows", bool_t, 0, "detect shadow", False)
gen.add("bilateral_d", int_t, 0, "d of bilateral", 11, 1, 200)
gen.add("bilateral_sigma_color", double_t, 0, "sigma color of bilateral", 40, 0, 300)
gen.add("bilateral_sigma_space", double_t, 0, "sigma space bilateral", 200, 0, 300)

exit(gen.generate(PACKAGE, "jsk_perception", "BackgroundSubstraction"))
